<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Allportdata</class>
 <widget class="QWidget" name="Main_Allportdata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
    font:12px;
}
QLabel
{
color:white;
}QPushButton
{
font: 16px;color: rgb(192,192,192);background-color: rgb(0, 0, 0);border-top: 2px solid white;border-left: 2px solid white;
}</string>
  </property>
  <widget class="QPushButton" name="BTNDDU">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>76</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>DDU/ERM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTCN1">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>76</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>TCN1_1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTCN2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>110</y>
     <width>76</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>TCN1_2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTCN4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>210</y>
     <width>76</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>TCN2_2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTCN3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>160</y>
     <width>76</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>TCN2_1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTCU">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>310</y>
     <width>76</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>TCU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNRIOM">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>260</y>
     <width>76</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>RIOM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNOTHER1">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>460</y>
     <width>76</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>OTHER2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNOTHER2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>410</y>
     <width>76</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>OTHER1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNRCU">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>360</y>
     <width>76</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>RCU</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
